commerce-frontend-taglib/src/main/resources/META-INF/resources/mini_cart/MiniCart.es.js

```diff
@@ -472,6 +490,16 @@ Cart.STATE = {
 	pendingOperations: Config.array().value(
 		[]
 	),
+
+	/**
+	 * For each product in the cart,
+	 * the related object received from the endpoint
+	 * contains 2 ID's:
+	 *
+	 * @param productId The id of the product relative to the cart.
+	 * @param cpinstanceId The id of the product relative to the catalog.
```

Comment:
Same here: capital `i`

Reply:
@ethanbustad The field is coming out like this from the Product model of the Cart even though in that class the property is camel-cased correctly.

Reply:
Interesting.... it's probably a bug in the serializer -- since `getCPIns...` isn't really proper camel-casing....... Alright, let's leave it as-is for now then. Thanks!